logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MarkMessagesAsNotifiedUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.wire.kalium.logic.feature.message
+
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.MessageRepository
+import com.wire.kalium.logic.functional.Either
+
+class MarkMessagesAsNotifiedUseCase(private val messageRepository: MessageRepository) {
```

Comment:
I think here we could also extract an interface and have the implementation internal, so we can benefit from scoping the implementation and later mock the interface on reloaded for tests :D